src/coreclr/jit/layout.cpp

```diff
@@ -421,22 +421,22 @@ void ClassLayout::InitializeGCPtrs(Compiler* compiler)
 }
 
 //------------------------------------------------------------------------
-// HasGCByRef: does the layout contain at least one GC ByRef
+// IsStackOnly: does the layout represent a block that can never be on the heap?
+//
+// Parameters:
+//   comp - The Compiler object
 //
 // Return value:
-//    true if at least one GC ByRef, false otherwise.
+//    true if the block is stack only
 //
-bool ClassLayout::HasGCByRef() const
+bool ClassLayout::IsStackOnly(Compiler* comp) const
 {
-    unsigned slots = GetSlotCount();
-    for (unsigned i = 0; i < slots; i++)
+    // Byref-like structs are stack only
```

Comment:
Is there a documentation that I can read more about this?

Reply:
Yep, e.g. https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/ref-struct Basically, Span itself is the most popular example